### DISCUSSION TITLE
Question regarding the delay function

### DISCUSSION THREAD

**Discussion by PwccaCode** [taskforcesh/bullmq#2306] [Q&A]
Using large delays should introduce higher cpu usage?

im trying to create around 500 jobs with close to 115600 ms delay.

and after the entire queue has been consumed and all the jobs are in redis waiting to be processed i have very high cpu usage until the jobs are done.

**Comment by manast** [comment]
The duration of the delay has no effect on performance. 500 jobs could be processed on a dummy processor in less 100ms (depending on the CPU type, concurrency, etc), so I doubt BullMQ is the origin of the high CPU load.
